### PR TITLE
KOGITO-6239 - kogito-quarkus-common-deployment artifact is missing in…

### PR DESCRIPTION
**Pull request by MarianMacik**
… kogito-bom

[KOGITO-6239](https://issues.redhat.com/browse/KOGITO-6239)

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run all builds</b>  
  Please add comment: <b>Jenkins retest this</b>

* <b>Run (or rerun) specific test(s)</b>  
  Please add comment: <b>Jenkins (re)run [kogito-runtimes|optaplanner|kogito-apps|kogito-examples|optaplanner-quickstarts|optaweb-employee-rostering|optaweb-vehicle-routing] tests</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Run (or rerun) LTS specific test(s)</b>  
  Please add comment: <b>Jenkins (re)run [kogito-runtimes|optaplanner|kogito-apps|kogito-examples|optaplanner-quickstarts|optaweb-employee-rostering|optaweb-vehicle-routing] LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

* <b>Run (or rerun) native specific test(s)</b>  
  Please add comment: <b>Jenkins (re)run [kogito-runtimes|optaplanner|kogito-apps|kogito-examples|optaplanner-quickstarts|optaweb-employee-rostering|optaweb-vehicle-routing] native</b>

* <b>Full Kogito testing</b> (with cloud images and operator BDD testing)  
  Please add comment: <b>Jenkins run BDD</b>  
  <b>This check should be used only if a big change is done as it takes time to run, need resources and one full BDD tests check can be done at a time ...</b>
</details>

### CODE REVIEW

**Review by winklerm — APPROVED** [review]
Thanks!

File: kogito-bom/pom.xml [hunk 1]
```diff
@@ -2128,6 +2128,11 @@
         <version>${project.version}</version>
         <classifier>javadoc</classifier>
       </dependency>
+      <dependency>
+        <groupId>org.kie.kogito</groupId>
+        <artifactId>kogito-quarkus-common-deployment</artifactId>
+        <version>${project.version}</version>
+      </dependency>
```
Comment (danielezonca) [inline kogito-bom/pom.xml:2135]: Please add also `sources`
```suggestion
      <dependency>
        <groupId>org.kie.kogito</groupId>
        <artifactId>kogito-quarkus-common-deployment</artifactId>
        <version>${project.version}</version>
      </dependency>
      <dependency>
        <groupId>org.kie.kogito</groupId>
        <artifactId>kogito-quarkus-common-deployment</artifactId>
        <version>${project.version}</version>
        <classifier>sources</classifier>
      </dependency>
```

Comment (MarianMacik) [reply kogito-bom/pom.xml:2135]: @danielezonca Other -deployment modules don't have them, should I still add sources just for this common module?

Comment (danielezonca) [reply kogito-bom/pom.xml:2135]: Ok this is quite strange but if so fine for me to not declare it for now and maybe review this with another ticket

**Review by danielezonca — APPROVED** [review]

**Comment by MarianMacik** [comment]
Every job succeeded at least on one platform, so I think it's OK.

**Comment by MarianMacik** [comment]
Jenkins rerun kogito-examples tests

**Pull request merged**